core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -117,6 +117,7 @@
 
   {# TODO(bhenning): Remove these once RouterServices is decoupled from the exploration editor #}
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ChangeListService.js"></script>
```

Comment:
Is it needed here?

Reply:
Doesn't collection editing come along with exploration editing? I'm not fully clear about the difference between these two functions.

Reply:
I double checked it. It's needed. If I removed this import line, collection editor page will crash.